frontend/src/app/moduleDisplay.tsx

```diff
@@ -115,34 +117,28 @@ class ModuleDisplay extends Component<any, any, any> {
         );
     }
 
-    private updateDate = (date, type, path) => {
-        if (type === "draft") {
+    private updateDate = (draftDate,releaseDate,releaseVersion) => {
             this.setState({
-                draftUpdateDate: date,
+                draftUpdateDate: draftDate,
                 // tslint:disable-next-line: object-literal-sort-keys
-                draftPath: path
+                releaseUpdateDate: releaseDate,
+                releaseVersion: releaseVersion                
             });
-        }
-        else {
-            this.setState({
-                releaseUpdateDate: date,
-                // tslint:disable-next-line: object-literal-sort-keys
-                releasePath: path
-            });
-        }
-
-    };
-
+    }
+    
     private fetchModuleDetails = (data) => {
-        this.setState({ modulePath: data.location.pathname })
+        this.setState({
+            modulePath: data.location.pathname, 
+            releasePath: "/content" + data.location.pathname + ".preview"
+        })
 
         fetch(data.location.pathname + '.4.json')
             .then(response => response.json())
             .then(responseJSON => {
                 // console.log('fetch results:',responseJSON["en_US"])
                 this.setState({
                     moduleTitle: responseJSON.en_US["1"].metadata["jcr:title"],
-                    resourceType: responseJSON["sling:resourceType"],
+                    moduleType: responseJSON.en_US["1"].metadata["pant:moduleType"],
```

Comment:
As per our discussion, we will be keeping this as a hard coded value "1", assuming 2 thinngs:
1. the version "1" is the first draft and it will always exist.
2. the module type remains the same throughout all the versions.

Reply:
Please don't forget to add the `// FIXME` tag that we talked about:
- If the module type is the same throughout all versions, it should exist on the `module`, not the `moduleVersion`
- If the module type is NOT the same throughout all versions, we should be reading it off the latest version
However we do not yet have that information, so the decision is to leave the functionality as-is for now.

Reply:
Sure Ben!
I will take care of this and put the explanation under the "FIXME" tag